app/models/vm_or_template/operations.rb

```diff
@@ -75,16 +83,7 @@ def rename_queue(userid, new_name)
       :userid => userid
     }
 
-    queue_opts = {
-      :class_name  => self.class.name,
-      :method_name => 'rename',
-      :instance_id => id,
-      :role        => 'ems_operations',
-      :zone        => my_zone,
-      :args        => [new_name]
-    }
-
-    MiqTask.generic_action_with_callback(task_opts, queue_opts)
+    run_command_via_queue(task_opts, :method_name => "rename", :args => [new_name])
```

Comment:
This looks like a typo, should have been `run_command_via_task`, right?

The params do match the `_task` signature, not `_queue`,
and so does the code (`MiqQueue.put` vs `MiqTask.generic_action_with_callback`).

And, so does UI travis ;)... https://travis-ci.org/ManageIQ/manageiq-ui-classic/jobs/615790473#L2060 

(
```diff
-    run_command_via_queue(task_opts, :method_name => "rename", :args => [new_name])
+    run_command_via_task(task_opts, :method_name => "rename", :args => [new_name])
```
passes, creating a PR)